test/test-operation.js

```diff
@@ -32,260 +32,638 @@ var helpers = require('./helpers');
 var Sway = helpers.getSway();
 
 describe('Operation', function () {
-  var swaggerApi;
+  context('swaggerApiRelativeRefs', function () {
```

Comment:
Yeah, let's move it to `test-issues.js` and I'll merge it.  Having a helper makes sense.  Also, this is the only use of `context` as well so maybe we can do a per-test invocation of your helper?